Dockerfile

```diff
@@ -28,6 +28,12 @@ RUN useradd --create-home --shell /bin/bash $USER && \
 
 # enter app directory
 WORKDIR /home/$USER/app
+# switch to non-root $USER
+USER $USER
```

Comment:
Part of the idea of delaying the switch in `USER` was to copy the app source files using root; so they are readable but not writeable by the container process.

Should we be concerned about that?